### DISCUSSION TITLE
System Service/Daemon on OpenWRT?

### DISCUSSION THREAD

**Discussion by ringtailedfox** [bluenviron/mediamtx#2576] [Questions]
### Question

I'd love the ability to configure if MediaMTX starts with my OpenWRT router or not.  I think this could be a useful feature, instead of having to manually start the program any time i'd like to stream to my LAN (or beyond)...

Is there a way to do this?  I believe OpenWRT still uses procd instead of sysvinit or systemd (to conserve space)...

Please let me know if such a task is possible!

My current version of OpenWRT is below:

BusyBox v1.35.0 (2023-01-03 00:24:21 UTC) built-in shell (ash)

<html>
<body>

Model | Netgear Nighthawk X4S R7800
-- | --
Architecture | ARMv7 Processor rev 0 (v7l)
Target Platform | ipq806x/generic
Firmware Version | OpenWrt 22.03.3 r20028-43d71ad93e / LuCI openwrt-22.03 branch git-23.292.78392-9f66674
Kernel Version | 5.10.161

</body>
</html>

**Comment by aler9** [comment]
Hello, i've added to the README instructions on how to create a OpenWrt service:

https://github.com/bluenviron/mediamtx#openwrt

[a comment by an automated account is not shown]
